index.html

```diff
@@ -544,6 +561,7 @@ <h2>
 					<label for="data">Base64: </label>
 					<input type="text" id="data" style="width:60vw" value="gaVoZWxsb6V3b3JsZA==" />
 					<button onclick="parseBase64()">Parse</button>
+					<label><input type="checkbox" id="chk_beautify" onchange="beautify()"> beautify</label>
```

Comment:
I wonder if the label should read `indent` instead of `beautify`

When I see `beautify` I also expect that it would colour the response like on http://jsbeautifier.org/